tests/features/support/step-definitions.js

```diff
@@ -99,6 +99,7 @@ defineStep('I should not be on (the ){target}', (pageName) =>
 defineStep('I wait for (a/an/the)( ){target}', (qa) =>
   i.waitForTargetToBeVisible(qa));
 
+// TODO: remove if redundant
```

Comment:
There don't appear to be any references to this anymore. I think it can be removed with this already.